### PR TITLE
Compiler/CLI usability improvements

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: docs/compiler.md [hunk 1]
```diff
@@ -133,18 +143,25 @@ Toolkit](https://github.com/neo-project/neo-blockchain-toolkit/). To do that
...
+specify manifest output file with `-m`:
+```
+./bin/neo-go contract compile -i mycontract.go -c myconfig.yml -m mycontract.manifest.json
+```
 
+Example YAML file contents:
 ```
+name: Contract
 supportedstandards: []
 events:
```
Comment (fyrchik) [inline docs/compiler.md:166]: `safemethods`?

File: docs/compiler.md [hunk 2]
```diff
@@ -153,10 +170,10 @@ events:
         type: ByteString
 ```
 
-It's passed to the `deploy` command via `-c` option:
+Then the manifest can be passed to the `deploy` command via `-m` option:
 
 ```
-$ ./bin/neo-go contract deploy -i contract.nef -c contract.yml -r http://localhost:20331 -w wallet.json -g 0.001
+$ ./bin/neo-go contract deploy -i contract.nef -m contract.manifest.json -r http://localhost:20331 -w wallet.json
```
Comment (fyrchik) [inline docs/compiler.md:176]: We should make file names unified with the previous command (either both `contract*` or `mycontract*`).

**Review by AnnaShaleva — APPROVED** [review]

File: docs/compiler.md (hunk 1 above)
Comment (roman-khimov) [reply docs/compiler.md:167]: Added that.

File: docs/compiler.md (hunk 2 above)
Comment (roman-khimov) [reply docs/compiler.md:177]: Not sure it's really important, but I've changed everything to `contract*`.

**Review by fyrchik — APPROVED** [review]

**Pull request merged**